ops/Makefile

```diff
@@ -35,9 +49,62 @@ deploy-%-api: .valid-env-% api.tfvars
 	terraform -chdir=$* apply -auto-approve -var-file=../api.tfvars -target="module.simple_report_api"
 
 promote-%: .be-logged-in .valid-env-%
-	az webapp deployment slot swap -g prime-simple-report-$* -n simple-report-api-$* --slot $(SOURCE_SLOT) --target-slot $(TARGET_SLOT)
+	az webapp deployment slot swap -g prime-simple-report-$* -n $(API_NAME) --slot $(SOURCE_SLOT) --target-slot $(TARGET_SLOT)
+
+wait-for-%-slot-commit: .valid-env-%
+	@echo "Starting wait for commit $(DEPLOYED_COMMIT) in $(SOURCE_SLOT) slot of $* at `date`"
+	until  [[ "$(DEPLOYED_COMMIT)"  == "`$(CURL) $(SLOT_INFO_ENDPOINT) | jq -r .git.commit.id`" ]] ; do sleep 5; done;
```

Comment:
How likely is it that this condition could never be satisfied? (Ditto for all of the new targets)

Reply:
If the app does not deploy successfully, this condition _should_ not be satisfied ever (that's why it's wrapped in a timeout everywhere we call it in the deploy jobs). The reason for the readiness checks is that this condition is sometimes satisfied when it shouldn't be, so the current deploy sequence is:
1. tell the app service to pick up a new image
2. wait for the expected image to be deployed (verifying it either by commit or by release tag)
3. wait for the current deployment to be ready to receive requests

That last step is what we were missing before.

Reply:
Ahh, I had missed the timeouts earlier.